images/api/App.js

```diff
@@ -46,6 +54,13 @@ class App {
   }
 
   async start() {
+    //Initiate consumer-scripts for listening
+    fs.readdirSync(consumers).forEach(file => {
+      require(consumers + file);
```

Comment:
Is dit niet op te lossen door een simpele index file voor de consumers. en die dan uit de helpers te halen?

Reply:
Is dat een meerwaarde over automatic require-ing?

Reply:
ni echt, gewoon common practice.